packages/animations/browser/src/dsl/animation_transition_factory.ts

```diff
@@ -123,12 +123,16 @@ function checkNonAnimatableInTimelines(
     return;
   }
 
+  const propsToIgnore = new Set<string>(['easing']);
```

Comment:
Can we maybe rename this to something like `nonAnimatableProps` or update the set to add a comment on each property explaining why it's ignored?
```suggestion
  const propsToIgnore = new Set<string>([
      'easing', // easing is not animated, but rather is a property of the animation
  ]);
```

Reply:
yes I can definitely add a comment there 🙂 

regarding the renaming I am not completely sure, because non-animatable props are props which we actually want to show warnings for when the user tries to animate them, this set here instead is a set of non-animatable props which we want to ignore in the non-animatable check (meaning that they are not animatable but safe to be used as animatable)

mh.... maybe something like `allowedNonAnimatableProps`?

Reply:
Adding the comment and keeping the variable name the same works for me :)

Reply:
@atscott I've rename the set to `allowedNonAnimatableProps` since it actually does feel much more clear than propsToIgnore` I hope that's fine

I've also added the comment for `easing` I might have been a bit verbose in it, please let me know what you think 🙂

Reply:
👌